autoload/neomake/makers/ft/java.vim

```diff
@@ -348,5 +365,26 @@ function! s:GetGradleClasspath() abort
     return ''
 endf
 
+function! s:GetAntClasspath() abort
+    let ant = s:findFileInParent('build.xml', expand('%:p:h', 1))
```

Comment:
Could use `neomake#utils#FindGlobFile` - but that's more an issue of the java module itself.

Reply:
Yeah, I tried to follow the original methods.